### PR TITLE
Standardizes chef dispenser chems

### PR DESCRIPTION
**Pull request by MintyPhresh**
[Chemistry][QoL][IDK???]
## About the PR
Different maps have different versions of the chef's dispenser-- Some, like Cog1, have a version that's a slightly modified soda dispenser. Others, like Kondaru, have a custom one that only carries condiments and milks. This PR combines the two, replaces all old iterations with this combination, and adds the new dispenser to Clarion, Cog2, Atlas, and Oshan, which were all missing either iteration before.

## Why's this needed?
Standard equipment between maps lets people make the same foods and flavors regardless of what map they're playing on.

### CODE REVIEW

**Comment by TheTyrant25** [comment]
this is now the 4th time this pr has been made i think

**Comment by TDHooligan** [comment]
i have been really pushing to get food reagents from botany regularly since the last PR. and have only received some once as a QM (apple juice), after waiting 65 minutes

i am going to keep trying but i still *really* think this is a good pr

**Comment by MintyPhresh** [comment]
> this is now the 4th time this pr has been made i think

Can't find anything that's exactly the same, although something similar happened with the two bar dispensers #15172

**Comment by TDHooligan** [comment]
#17062
#6363
then once before iirc

**Comment by MintyPhresh** [comment]
Terrible news. No idea what about a map's design makes the dispenser only acceptable on some maps... If the chems shouldn't be available on any map then I can just change the PR to not include the chem.

**Comment by MintyPhresh** [comment]
#3587 was the first one as far as I can tell

**Comment by MintyPhresh** [comment]
Yeah, I made a forum post for this. Closing this while the discussion happens.
Thread [here.](https://forum.ss13.co/showthread.php?tid=22185)

**Comment by MintyPhresh** [comment]
whoops this should close it

**Comment by MintyPhresh** [comment]
Reopened after discussion in the [forum thread](https://forum.ss13.co/showthread.php?tid=22185&pid=200451#pid200451) died down.

**Comment by TheTyrant25** [comment]
DON'T LET THE FLAME DIE OUT

**Comment by ZeWaka** [comment]
x

**Comment by ZeWaka** [comment]
x

**Review by flrsh — CHANGES_REQUESTED** [review]
I think you probably already know my opinion from my comments on your previous PRs. I think it's fine if the chef has to run into the bar to use the soda fountain, and I think it's good that there are reagents that require botany / QM. I appreciate wanting to make the dispensers consistent though. I think I can accept mustard / ketchup for all maps, since right now it *is* annoying to run out of them in the squeeze bottles. However, please remove mint since it actually is a reagent with various recipes, and it's important that you need botany's help. Same with salt since it's important for wraiths and shouldn't be a one click thing.

**Comment by MintyPhresh** [comment]
Sorry, those two shouldn't have been in there.

**Review by Tarmunora — CHANGES_REQUESTED** [review]
something appears to have exploded with your last merge commit

**Comment by MintyPhresh** [comment]
Yeaaaah. Not sure what happened, was just trying to fix the merge conflicts. Working on it now

**Comment by MintyPhresh** [comment]
I think I fixed whatever exploded

**Comment by MintyPhresh** [comment]
Sorry about that erroneous review request. It exploded again

**Comment by ZeWaka** [comment]
!merge_upstream

**Comment by TobleroneSwordfish** [comment]
Sorry this has been sitting for so long, but after some discussion we've decided we don't want to put more botany produce in the kitchen for free and that a better route would be to standardize away from the expanded dispensers

File: code/modules/chemistry/Chemistry-Dispenser.dm [hunk 1]
```diff
@@ -523,11 +523,13 @@ TYPEINFO(/obj/machinery/chem_dispenser)
...
-	dispensable_reagents = list("cola", "juice_lime", "juice_lemon", "juice_orange", "mint", "mustard", "pepper", \
-								"juice_cran", "juice_cherry", "juice_pineapple","coconut_milk", "ketchup", \
-								"sugar", "water", "vanilla", "tea", "chocolate", "chocolate_milk","strawberry_milk")
+	name = "HAPPY CHEF Dispense-o-tronic"
+	desc = "It's covered in a thin layer of acrid-smelling dust. The contents probably taste more like preservatives than whatever they're supposed to be."
+	dispensable_reagents = list("juice_cherry", "chocolate", "chocolate_milk", "coconut_milk", \
+								"cola", "juice_cran", "ginger_ale", "gravy", "grenadine", "ketchup", \
+								"juice_lemon", "juice_lime", "mustard", "juice_orange", \
+								"pepper", "juice_pineapple", "strawberry_milk", "sugar", \
+								"tea", "juice_tomato", "vanilla")
```
Comment (Tarmunora) [inline code/modules/chemistry/Chemistry-Dispenser.dm:532]:
```suggestion
	dispensable_reagents = list("ketchup","mustard","salt","pepper","gravy","chocolate","mint","chocolate_milk","strawberry_milk")
```

**Comment by Tarmunora** [comment]
on second though, let's just standardize to the Nadir list

**Comment by MintyPhresh** [comment]
Either's fine for me. Frankly at this point I just want it closed, merge or no merge

File: code/modules/chemistry/Chemistry-Dispenser.dm (hunk 1 above)
Comment (MintyPhresh) [reply code/modules/chemistry/Chemistry-Dispenser.dm]: Standardizing to the Nadir list readds mint and salt which I was specifically asked to remove earlier. If that decision's changed, that's great, just letting you know

**Pull request merged**